docs/source/notebooks/sampling_conjugate_step.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>Maybe explain why <code>point[self.conc_prior.transformed.name]</code> is equal to <code>conc_prior</code>? In particular why we have to use the <code>name</code> attribute?</p><p>Also, if possible, some type hints could help understanding</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Getting the type hints for the input PyMC3 variables was a little tricky so I only did them for the obvious inputs (<code>counts</code>  and <code>point</code>).  I added some text to explain the long piece you mentioned.</p>

Reply:
<p>I don't seem to see the text you added -- did you commit it?</p>